src/interpreter/ByteCode.h

```diff
@@ -1144,20 +1154,24 @@ struct GetObjectInlineCacheData {
 typedef Vector<GetObjectInlineCacheData, CustomAllocator<GetObjectInlineCacheData>, ComputeReservedCapacityFunctionWithLog2<>> GetObjectInlineCacheDataVector;
 
 struct GetObjectInlineCacheComplexCase {
-    GetObjectInlineCacheComplexCase()
+    GetObjectInlineCacheComplexCase(ObjectStructurePropertyName propertyName)
+        : m_propertyName(propertyName)
     {
     }
 
     void* operator new(size_t size);
     void* operator new[](size_t size) = delete;
 
     GetObjectInlineCacheDataVector m_cache;
+
+    ObjectStructurePropertyName m_propertyName;
 };
 
 struct GetObjectInlineCacheSimpleCase : public gc {
-    GetObjectInlineCacheSimpleCase()
+    GetObjectInlineCacheSimpleCase(ObjectStructurePropertyName propertyName)
+        : m_propertyName(propertyName)
     {
-        memset(this, 0, sizeof(GetObjectInlineCacheSimpleCase));
+        memset(m_cachedStructures, 0, sizeof(ObjectStructure*) * inlineBufferSize);
```

Comment:
This change initializes only the `m_cachedStructures` with 0, not including the `m_cachedIndexes` array.
Initialization of `m_cachedIndexes` is not necessary?

Reply:
Yes. we should init m_cachedStructures only